papas/reconstruction/PapasManager.h

```diff
@@ -5,6 +5,7 @@
 #include "papas/detectors/Detector.h"
 #include "papas/graphtools/DefinitionsNodes.h"
 #include <list>
+#include <string>
```

Comment:
is that necessary? 
it seems that the new include is not used in this header file.